docs(migration): add info about default updateEventTrigger change (#1393)
camunda/camunda-docs-manual: content/webapps/cockpit/bpmn/process-instance-migration.md

To set the [update event trigger flag]({{< ref "/user-guide/process-engine/process-instance-migration.md#events" >}}) for events, first create a mapping for the event. Then click on the green checkmark indicator to open the details tooltip. The opened popup contains a checkbox to update the event trigger.

{{< note title="Heads Up!" class="warning" >}}
By default, the update event trigger flag in cockpit migration is set for activities with message references.

> That is, for activities containing message boundary event, message boundary interrupting event, message intermediate catch event and for the receive task activity.

Reviewer: ```suggestion
I.e., for interrupting/non-interrupting message boundary events, message intermediate catch events, and receive tasks.
```
